internal/pkg/cli/dialog/use_template.go

```diff
@@ -43,7 +43,7 @@ func (p *Dialogs) AskUseTemplateOptions(projectState *project.State, inputs temp
 	return dialog.ask(inputs)
 }
 
-func (d *useTmplDialog) ask(inputs input.StepsGroups) (useTemplate.Options, error) {
+func (d *useTmplDialog) ask(stepsGroups input.StepsGroups) (useTemplate.Options, error) {
 	// Load inputs file
 	if d.options.IsSet(inputsFileFlag) {
```

Comment:
Tu su zmeny v implementacii, ... trocha som to prehadzal a vyuzil `*Ext` struktury, asi je to lepsie pozriet ako cely subor.